synapse/lib/stormhttp.py

```diff
@@ -28,6 +25,15 @@ def addLibFuncs(self):
                 }
                 return HttpResp(info)
 
+    async def _httpPost(self, url, headers=None, json=None, body=None):
+        async with aiohttp.ClientSession() as sess:
+            async with sess.post(url, headers=headers, json=json, data=body) as resp:
```

Comment:
lets set content-type to json here if json is not None.  That way, callers can avoid having to specify that header.

Reply:
actually, if json is none and a data is provided; aiohttp makes the content type determination by the type of data.  if the data is binary, it will be a octet-stream; otherwise if data is a dictionary it will be form encoded.